code/gradle.properties

```diff
@@ -2,13 +2,13 @@ org.gradle.jvmargs=-Xmx2048m
 android.injected.testOnly=false
 org.gradle.configureondemand=false
 
-moduleProjectName=media
-moduleName=media
-moduleAARName=media-phone-release.aar
-moduleVersion=3.0.0
+moduleProjectName=edgemedia
+moduleName=edgemedia
+moduleAARName=edgemedia-phone-release.aar
+moduleVersion=1.0.0
 
-mavenRepoName=AdobeMobileMediaSdk
-mavenRepoDescription=Android Analytics Extension for Audio and Video.
+mavenRepoName=AdobeMobileEdgeMediaSdk
+mavenRepoDescription=Android Edge Media Extension.
```

Comment:
Adobe Experience Platform Edge Media extension for the Adobe Experience Platform Mobile SDK